docs/pages/docs/apis/access-control.mdx

```diff
@@ -20,244 +20,171 @@ export default config({
+List-level access control can be specified in three ways, of increasing power.
 
-```typescript
-import { config, createSchema, list } from '@keystone-next/keystone';
-
-export default config({
-  lists: createSchema({
-    ListKey: list({
-      // Concise access control definition
-      access: true,
-      // Verbose access control definition
-      access: {
-        create: true,
-        read: true,
-        update: true,
-        delete: true,
-      },
-    }),
-  }),
-});
-```
+- `operation` access control lets you check the information in the `context` and `session` objects to decide if the
+  user is allowed to access the list.
+- `filter` access control lets you provide a GraphQL filter which defines the items the user is allowed to access.
+- `item` access control lets you write a function which inspects the provided input data and the existing object (if it exists)
+  and make a decision based on this extra data.
 
-When using verbose syntax, any operation which is not specified will default to `true`.
-
-The examples below will all use the concise syntax, however the various access control rules can all be applied using verbose syntax.
-
-There are three different ways you can specify access-control rules: **static**, **declarative**, and **imperative**.
-
-### Static (list)
-
-Static access control rules are simple boolean values.
-A value of `true` indicates that all users can perform the operation.
-A value of `false` indicates that no users can perform the operation.
-
-A static value of `false` implies that the operation can never be executed.
-As such, Keystone will exclude the related operations and types from the GraphQL API.
-For example, if you set `{ create: false }` then the mutations `createItem` and `createItems` will be removed from the GraphQL API.
-If you want to keep the operations in the GraphQL API while preventing all access, you can use the [imperative](#imperative-list) access control rule `() => false`.
-The excluded operations can still be access by using [`context.sudo()`](./context#new-context-creators).
-
-```typescript
-import { config, createSchema, list } from '@keystone-next/keystone';
-
-export default config({
-  lists: createSchema({
-    ListKey: list({
-      // Static access control definition
-      access: true,
-    }),
-  }),
-});
-```
+If access is **denied** due to any of the access control methods, the following responses will be returned from the GraphQL API:
 
-### Declarative (list)
+- **Mutations**
+  - **Single** operations return `null` and an access denied error.
+  - **Multi** operations return a data array with `null` values for the items which have access denied.
+    Error responses are returned for each `null` item.
+- **Queries**
+  - **Single** item queries return `null` with no errors.
+  - **Many** item queries filter out items which have access denied. No error responses are returned.
```

Comment:
So unlike multi mutations this will be an array without nulls? Probs not relevant to these docs, but is there reasons for this?

Reply:
That's correct. The general principle here is "queries never have errors. access denied should look identical to objects not existing".

So with that principle in mind, a `many` query where some items are access denied, the expected result is an array where those items simply don't show up at all.

It's probably worth us adding that general principle somewhere up the top of the document, as it helps inform each of the specific behaviours described in the details.

Reply:
Pls add this principle. Would be super helpful